src/praisonai/praisonai/cli_backends/__init__.py

```
@@ -1,73 +1,35 @@
-            f"cli_backend must be string, dict, or instance, got: {type(value).__name__}"
-        )
+import sys as _sys
+
+import praisonai_code.cli_backends as _cli_backends
+
+# Alias submodules so ``praisonai.cli_backends.<sub>`` resolves to the moved module.
+for _name in ("registry", "claude"):
+    try:
+        _mod = __import__(f"praisonai_code.cli_backends.{_name}", fromlist=[_name])
+    except Exception:  # pragma: no cover - optional submodule
+        continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)